frontend/src/app/revisions.test.tsx

```diff
@@ -58,6 +60,35 @@ describe('Revisions tests', () => {
         expect(dataListItemRow.exists()).toBe(true)
     });
 
+    it('should render a DataListToggle', () => {
+        const wrapper = mount(<Revisions {...props} />);
+        const dataListToggle = wrapper.find(DataListToggle);
+        expect(dataListToggle.exists()).toBe(true)
+    });
+
+    it('should render a DataListContent', () => {
+        const wrapper = mount(<Revisions {...props} />);
+        const dataListContent = wrapper.find(DataListContent);
+        expect(dataListContent.exists()).toBe(true)
+    });
+
+    // it('should render a Dropdown', () => {
```

Comment:
if we don't need these tests lets just delete them instead of commenting it out.

Reply:
done